Makefile.am

```diff
@@ -295,6 +295,47 @@ libeos_updater_flatpak_installer_libeos_updater_flatpak_installer_@EUFI_API_VERS
 # eos-updater-flatpak-installer program
 libexec_PROGRAMS += eos-updater-flatpak-installer/eos-updater-flatpak-installer
 
+###############################################################################
+# Introspection
+-include $(INTROSPECTION_MAKEFILE)
+INTROSPECTION_GIRS =
+INTROSPECTION_SCANNER_ARGS =
+INTROSPECTION_COMPILER_ARGS =
+AM_DISTCHECK_CONFIGURE_FLAGS += --enable-introspection
+EXTRA_DIST += m4/introspection.m4
+
+if HAVE_INTROSPECTION
+libeos-updater-flatpak-installer/Libeufi-@EUFI_API_VERSION@.gir: libeos-updater-flatpak-installer/libeos-updater-flatpak-installer-@EUFI_API_VERSION@.la
+libeos_updater_flatpak_installer_Libeufi_@EUFI_API_VERSION@_gir_INCLUDES = GObject-2.0 Gio-2.0
+libeos_updater_flatpak_installer_Libeufi_@EUFI_API_VERSION@_gir_CFLAGS = \
+	$(libeos_updater_flatpak_installer_libeos_updater_flatpak_installer_@EUFI_API_VERSION@_la_CFLAGS) \
```

Comment:
I think this should be:
```
$(STD_CFLAGS) \
$(EOS_UPDATER_FLATPAK_INSTALLER_CFLAGS) \
$(NULL)
```
rather than using the full list of `CFLAGS` from the library, otherwise you’ll end up with warnings and code coverage enabled for the GIR scanner, which will probably break things. GIR generates a C file which it compiles and runs to extract the runtime `GType` information from a library. Since it’s generated code, compiling it with warnings enabled is generally a bad idea.